app/src/androidTest/java/de/test/antennapod/ui/PreferencesTest.java

```diff
@@ -46,7 +45,6 @@
 import static de.test.antennapod.EspressoTestUtils.waitForView;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static junit.framework.TestCase.assertTrue;
-import static org.hamcrest.Matchers.allOf;
```

Comment:
I think re-organizing the imports throughout the whole project creates too much noise. I will add a checkstyle rule for import order etc at some point but at the moment, this is not too important.

Reply:
Well, it's not actually reorganizing, it's just getting rid of imports that aren't being used. Actual re-organizing looks much messier.